packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -1070,28 +1070,50 @@ def _parse_work_item_from_email(self):
             # pylint: disable=undefined-loop-variable
             self.set_work_item_variable(keys[0], parsed_email)
 
-    def _start_suite(self, data, result):
+    def _start_suite(self, *_):
```

Comment:
```suggestion
    def _start_suite(self, *args, **kwargs):
```

I guess this would be the Python way of doing it